examples/ansible.cfg

```diff
@@ -492,3 +492,30 @@
 
 # Set how many context lines to show in diff
 #context = 3
+
+[galaxy]
+# Controls whether the display wheel is shown or not
+#display_progress=
+
+# Validate TLS certificates for Galaxy server
+#ignore_certs = False
+
+# Role or collection skeleton directory to use as a template for
+# the init action in ansible-galaxy command
+#role_skeleton=
+
+# Patterns of files to ignore inside a Galaxy role or collection
+# skeleton directory
+#role_skeleton_ignore="^.git$", "^.*/.git_keep$"
+
+# Galaxy Server URL
+#server=https://galaxy.ansible.com
+
+# A list of Galaxy servers to use when installing a collection.
+#server_list=
+
+# GitHub personal access token
+#token=
```

Comment:
I think we should stop documenting this option, it's only used for `ansible-galaxy role login` and the working of the config value may make people think it's a token for Galaxy itself.

Reply:
OK.